### ISSUE TITLE
[xas_viewer] features requests on selecting E0

### ISSUE THREAD

**Issue by maurov** [xraypy/xraylarch#295]
@newville I have a couple of features requests from our users at the beamline, regarding E0 selection:

- [ ] select E0 at a set fraction of the the edge step.
- [ ] plot second derivative.
- [ ] select E0 at zero-crossing of the 2nd derivative.

I write down this here as a reminder.

**Comment by newville** [comment]
@maurov  All fine suggestions.  Will try those

**Comment by newville** [comment]
@maurov this is now fixed, almost ready to push out 0.9.55....

**Comment by maurov** [comment]
@newville thanks for working on these features. I see the plot of the second derivative, but I do not see in the GUI how to select E0 at a fraction of the edge step or at the zero-crossing of the 2nd derivative.

**Comment by newville** [comment]
@maurov yeah, sorry I should have commented here earlier: the second derivative was easy, "fraction of edge step" seemed harder -- but is still on the ToDo list.  Basically, I felt like I needed to get 0.9.55 out sooner rather than later.

So, I would say let's leave this open as it is not complete. Yet.
